src/renderer/main.js

```diff
@@ -1,8 +1,34 @@
+/**
+ * Perf optimization: Hook into require() to modify how certain modules load:
+ *
+ * - `lodash/merge` (used by `material-ui`) causes 119 require() calls at startup,
+ *   which take ~100ms. Replace it with `lodash.merge` which is equivalent.
+ *   See: https://github.com/callemall/material-ui/pull/4380#issuecomment-250894552
+ *
+ * - `inline-style-prefixer` (used by `material-ui`) takes ~40ms. It is not
+ *   actually used because auto-prefixing is disabled with
+ *   `darkBaseTheme.userAgent = false`. Return a fake object.
+ */
+let Module = require('module')
+const _require = Module.prototype.require
+Module.prototype.require = function (id) {
+  if (id === 'lodash/merge') id = 'lodash.merge'
+  if (id === 'inline-style-prefixer') return {}
+  return _require.apply(this, arguments)
+}
+
```

Comment:
This is quite the hack, but it's pretty safe since we're just intercepting the exact two modules listed there. And it improved startup time a pretty decent amount.